set.mm

```diff
@@ -435480,14 +435479,14 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 */
 htmldef "CMod" as "CMod";
   althtmldef "CMod" as "&#8450;Mod";
-  latexdef "CMod" as "{\mathbb{C}\mathrm{Mod}}";
```

Comment:
Here and below: the braces are indeed useless, but they show in a tex file that it comes from a single token, so could be useful to some scripts reading the TeX.

Reply:
Ok, so I guess I put them back? Btw `latex_collector.py` writes every LaTeX definition separately in each line, so it doesn't need braces to notice that it comes from a single token, also there are no spaces in each of those substitutions in this pr, which I guess is good enough confermation that they come from a single definition since I doubt a script wouldn't separate them with a whitespace or something else.

Reply:
> I doubt a script wouldn't separate them with a whitespace or something else.

You cannot make such an assumption.